### ISSUE TITLE
more general filtering (e.g. band pass filters)

### ISSUE THREAD

**Issue by apatlpo** [serazing/xscale#14]
I've got the feeling this is not implemented yet, while `scipy.signal.firwin` allows in principle.
Does anybody confirm?

**Comment by serazing** [comment]
Yes, you can only do a low-pass filtering for the moment. I usually estimate the high-pass component by subtracting the low-pass component to the original field. Implementing high-pass filtering would just require to set 1 - low-pass kernel. The band-pass would require to make a composition of a high-pass and a low pass-kernel. Feel free to modify the code in `xscale.window.linearfilters.Window.set` and submit a pull request if you want this functionality added to `xscale`.

**Comment by mada0007** [comment]
Hello I would be glad if someone can kindly assist me on installing and loading xscale on mac. I have installed following the steps online and yet when import it doesnt work. Thanks for your kindness in advance.

**Comment by stephanieleroux** [comment]
hi Guillaume, how about a lanczos "window" like you had added in oocgcm?
